src/index.css

```diff
@@ -0,0 +1,420 @@
+/* PRIMARY GLOBAL FONT */
+@import url('https://fonts.googleapis.com/css2?family=Lato&family=Montserrat&family=Poppins:wght@300;400;500;600;700&display=swap');
+
+/* SECONDARY GLOBAL FONT */
+@import url('https://fonts.googleapis.com/css2?family=Lato&family=Montserrat&family=Poppins:wght@300;400;500;600;700&family=Roboto+Slab&display=swap');
+
+/* *** GLOBAL STYES *** */
+:root {
+  /* shades of primary color */
+  --primary-white-1: #fafafa;
+  --primary-white-2: #fff;
+  --primary-blue: #0290ff;
+  --primary-black: #121212;
+  --primary-gray: #888;
+  --secondary-font: 'Roboto Slab', serif;
```

Comment:
- [ ] Weldone for your effort on this task. However, I think you can improve your styling to align with the model design on Zeplin by fixing some elements. 
- First, the grey background color you choose is darker than the background color on the design guide. I will suggest you use a color picker like the eye dropper chrome extension to help you select the right colors.